compiler/optimizer/OMRSimplifierHelpers.cpp

```diff
@@ -346,8 +346,6 @@ TR::Node *foldRedundantAND(TR::Node * node, TR::ILOpCodes andOpCode, TR::ILOpCod
       {
       switch(constOpCode)
          {
-         case TR::sconst: case TR::cconst:
```

Comment:
I don't think this is right - didn't you mean to just delete the `cconst` case - the sconst is not being removed right?

Reply:
Sorry, that was a mistake, only `cconst` should be deleted. Fixed in c215804